packages/expo-speech/src/index.ts

```diff
@@ -0,0 +1,2 @@
+import * as Speech from './Speech/Speech';
```

Comment:
Remove this file and point `main` to `Speech/Speech` 🙂

Reply:
For explanation see discussion at https://github.com/expo/expo/pull/2920/files/7ea7e9f76e0a1f1c0b3d321493ac8d607293d049#diff-4b06aa1fcb81c39457d7f817740f39ab